src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = 40;
+  const total = days * dayPrice;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days < 3) {
+    return total;
+  } 
+  
+  if (days < 7) {
+    return total - smallDiscount;
```

Comment:
Similar to the previous comment, use a constant to represent the threshold for a 'big discount' instead of hardcoding the value 7. This aligns with checklist item 2.